onadata/apps/api/tests/viewsets/test_charts_viewset.py

```diff
@@ -160,6 +161,7 @@ def test_get_on_categorized_field(self):
         self.assertEqual(response.data['data'][0]['gender'], 'Male')
         self.assertEqual(response.data['data'][1]['gender'], 'Female')
 
+    @skip("https://github.com/onaio/onadata/pull/2170")
```

Comment:
@DavisRayM Not sure why we are skipping this

Reply:
Skipping it because we default to JSON instead of returning a bad request at the moment

Reply:
Can we just remove or edit that test?

Reply:
I can remove it, it's a bit similar to one of the other tests so I wasn't so sure about editing it. Let me remove it and re-request review

Reply:
cool